docs/API.md

```diff
@@ -1551,6 +1551,73 @@ Example: `GET ~bob/trusty/wordpress-42/meta/id-series`
 }
 ```
 
+#### GET *id*/meta/common-extra-info
+
+The meta/common-extra-info path reports any common metadata recorded for 
+the base entity. 
+This contains only information stored by clients - the API server itself
+does not populate any fields. The resulting object holds an entry for each
+piece of metadata recorded with a PUT to `meta/common-extra-info`.
+
+```go
+type CommonExtraInfo struct {
+        Values map[string] interface{}
+}
+```
+
+Example: `GET wordpress/meta/common-extra-info`
+         `GET precise/wordpress-32/meta/common-extra-info`
+
+```json
+{
+    "featured": true,
+    "vcs-digest": "4b6b3c7d795eb66ca5f82bc52c01eb57ab595ab2"
```

Comment:
a digest seems to me to be exactly the sort of thing that isn't common, could you find a better example? the one's you're adding might be good.